test/unit/abstract_test.rb

```diff
@@ -351,6 +351,40 @@ def self.create_option_builder
 
   end
 
+  describe 'defaults' do
+    class TestDefaultsCmd < HammerCLI::AbstractCommand
+      option '--test', 'TEST', '', :attribute_name => :different_attr_name
+      option ['--test-multi1', '--test-multi2'], 'TEST_MULTI', '', :attribute_name => :multiple_switches_option
+
+      def options
+        super
```

Comment:
The fact that I had to use this hack makes me think I should have extracted the code into a class to make it testable. I'll probably refactor that.